frontend/src/components/ingress/Details.tsx

```diff
@@ -36,6 +37,29 @@ export default function IngressDetails() {
     );
   }
 
+  /** this function is used to format the url to be used in the Link component */
+  function isFormatURL(url: any) {
+    if (url === '*') {
+      return '*';
+    }
```

Comment:
This could be removed since we'd catch it with the exception anyway, but exceptions should be avoided, so it's good! 👍

Reply:
Fixed it!